platform/ios/src/MGLMapView.mm

```diff
@@ -6877,11 +6894,15 @@ void deactivate() override
     }
 
 private:
-    __weak MGLMapView *nativeView = nullptr;
+    __weak MGLMapView* nativeView = nullptr;
 
     NSUInteger activationCount = 0;
 };
 
+void MBGLMapViewRenderable::bind() {
+    backend.restoreFramebufferBinding();
```

Comment:
By the way, it looks like the macOS implementation sets the viewport in the equivalent to this method instead of `restoreFramebufferBinding()`. Not sure if that’s significant.

Reply:
unified